client/i18n/locales/english/translations.json

```
@@ -52,6 +52,8 @@
     "create-post": "Create a help post on the forum",
     "cancel": "Cancel",
     "reset-lesson": "Reset this lesson",
+    "revert": "Revert",
+    "revert-to-saved-code": "Revert to saved code",
```

```suggestion
    "revert-to-saved-code": "Revert to Saved Code",
```

I think this matches the other button text.